### DISCUSSION TITLE
Hybrid-X1X3-G4- compability

### DISCUSSION THREAD

**Discussion by zeair** [wills106/homeassistant-solax-modbus#556] [General]
Hello, currently I'm using Domoticz to read data from my inverter using https://github.com/saidlm/Domoticz-Solax-plugin/blob/main/plugin.py 

I want to migrate to Home Assistant - would this integration work with this model ? 

Here is the inverter communication documentation - https://github.com/saidlm/Domoticz-Solax-plugin/blob/main/docs/Hybrid-X1X3-G4-ModbusTCPRTU-V3.21-English_0622-public-version.pdf

**Comment by wills106** [comment]
Yes it's fully supported, including control of the Inverter. I need to add in some of the new functionality a recent Firmware introduced, but all the basics are there and alot more than that domoticz plugin provides.

**Comment by zeair** [comment]
Thank you! It works and is amazing!
